doc/administrator/administrate.rst

```diff
@@ -181,3 +181,30 @@ to see (and edit, if applicable) the features within this area.
 * ``Roles``: Restricted to the following roles.
 * ``Read/write``: Allows the linked users to change the objects.
 * ``Area``: Active in the following area, if not defined it is active everywhere.
+
+Users
+-----
+
+Each user may have from 1 to n roles, but each user has a default role from
+which are taken the some settings. Currently the default role (defined through
+the "Settings from role" selection) has an influence on the extent retrieved
+through the role and some functionalities.
```

Comment:
not sure I understand... to which part of the sentence does "and some functionalities." refer to?

Reply:
Would that be clearer:

```
Currently the default role (defined through the "Settings from role" selection) has an
influence on the role extent and on some functionalities regarding their configuration.
```

Reply:
Yes, thanks